app/templates/editions/index.hbs

```diff
@@ -2,6 +2,8 @@
 <section class="container" aria-labelledby="ember-editions">
   <h1 id="ember-editions">Ember Editions</h1>
   <p>
+    An edition is a collection of features and patterns that represent
+    the best way to build Ember apps for a range of Ember versions.
```

Comment:
I think the paragraph, as is, will become long to read. The paragraph includes the word `edition` too many times, which may lessen the impact. The following screenshot is at a desktop resolution.

<img width="1020" alt="Screenshot 2021-09-02 at 07 51 27" src="https://user-images.githubusercontent.com/16869656/131789426-4f7a2244-b328-4724-b653-baf515102539.png">

I think the sentence that you wrote is a key point and would like to emphasize it in its own paragraph.

```suggestion
    <em>An edition is a collection of features and patterns that represent
    the best way to build Ember apps for a range of Ember versions.</em>
  </p>
  <p>
```

<img width="1020" alt="Screenshot 2021-09-02 at 07 52 43" src="https://user-images.githubusercontent.com/16869656/131789507-b01b90f8-bc36-453c-bc35-ddda4bc63c50.png">